### ISSUE TITLE
Question regarding geofences and http

### ISSUE THREAD

**Issue by lucadegasperi** [transistorsoft/react-native-background-geolocation#1498]
Hello,
This is not an issue but more of a question regarding the behavior of the plugin.
Here's my use case:
I use the plugin for both continuous tracking and geofencing.
When using the plugin in geofence mode only, I don't want any http requests to be made, therefore I set the `url` config param to an empty string.
When using the plugin for continuous tracking, I do set the `url` back to my server's endpoint. I also have `autoSync` always set to true which is the default for the plugin.
My question is the following: When switching to continuous tracking, are the locations obtained while in geofence mode sent to the server as well? How does the plugin work in this case? Should I manually flush the locations obtained while in geofence mode before starting the continuous tracking?

Thank you.

**Comment by christocracy** [comment]
>My question is the following: When switching to continuous tracking, are the locations obtained while in geofence mode sent to the server as well?

Yes.  What you want to do is change `Config.persistMode` (read the API docs about it).

Also read the API docs for "*HTTP Guide*", linked throughout.

- The HTTP Service feeds off of records in the plugin's SQLite database.
- `Config.persistMode` controls when records are inserted into SQLite.
- If no record is persisted, there can be no HTTP request for it.
